calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
+    total += grades[i];
+  }
+
+  let avg = total / grades.length;
```

Comment:
Maybe renaming "avg" to something like "gradeAvg" would make the variable name more descriptive.